Makefile

```diff
@@ -42,13 +42,13 @@ fmt:
 	@echo Running go fmt on query service ...
 	@$(GOFMT) -e -s -l -w .
 
-.PHONY: build-signoz-collector
-build-signoz-collector:
+.PHONY: build-and-push-signoz-collector
+build-and-push-signoz-collector:
```

Comment:
Can you add a new target to this build-and-push? When working locally it helps to have a separate target which just creates a new image.

Reply:
@srikanthccv Done.